cmd/minishift/cmd/config/validations.go

```diff
@@ -41,7 +41,7 @@ func IsValidDriver(string, driver string) error {
 }
 
 func RequiresRestartMsg(name string, value string) error {
-	api := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
+	api := libmachine.NewClient(constants.GetMinipath(), constants.MakeMiniPath("certs"))
```

Comment:
why can we not keep the _Minipath_ var? I know there are bootstrap issues which need to be worked around, but it seems odd to call a methods from the constant file - _constants.GetMinipath()_. It would also reduce the changeset considerably.

If this is not possible, we might want to consider moving _GetMinipath_ out of _constants.go_.